src/components/ArticleView.jsx

```diff
@@ -0,0 +1,15 @@
+import Box from "@mui/material/Box";
+
+const ArticleView = (props) => {
+  return (
+    <Box>
+      <h1>{props.article.title}</h1>
+      <h3>Author: {props.article.author}</h3>
+      <h4>Date: {props.article.created_at.split("T")[0]}</h4>
+      <img src={props.article.article_img_url} style={{ width: "100%" }} />
+      <p>{props.article.body}</p>
+    </Box>
```

Comment:
Consider here the ability of the user to navigate around your app, once you go to a single article there is no way to navigate back to the list or over to another page without using the back arrow on the browser. If you get time it would reflect well on your abilities with the Router to add in this additional navigational functionality 👍